src/backend/distributed/transaction/backend_data.c

```diff
@@ -92,6 +92,18 @@ typedef enum CitusBackendType
 	EXTERNAL_CLIENT_BACKEND
 } CitusBackendType;
 
+static const char *citusBackendPrefixes[] = {
```

Comment:
we usually start global variable names with a capital letter to easily distinguish them from local variables

Reply:
done, but tbh I never realized we had such a naming scheme, our naming scheme always seemed pretty arbitrary (example three liens below that uses 3 different types of casing for 4 different global variables)
```
static shmem_startup_hook_type prev_shmem_startup_hook = NULL;
static BackendManagementShmemData *backendManagementShmemData = NULL;
static BackendData *MyBackendData = NULL;
static CitusBackendType CurrentBackendType = CITUS_BACKEND_NOT_ASSIGNED;
```